### DISCUSSION TITLE
How to signout serverside?

### DISCUSSION THREAD

**Discussion by madeinspace** [nextauthjs/next-auth#2707] [Help]
I'm running into a usecase where I have a multitenant application and I need to automatically signout if the tenant change in the URL

say "myapp.com/tenant1" where tenant1 is already signed in, if he changes tenant1 to tenant2 in the URL I want to be able to signout tenant1 server side. Is that possible or do I need to remove the session cookie myself?

At the moment I'm doing it on the client like so but I'm getting a flash of non secured page before signout.

`// if (process.browser && session && session.user.name != tenantAlias) signOut();`

Thanks!

**Comment by balazsorban44** [comment]
you will have to delete the cookie, yes, and you cannot do this totally from the backend.

you might get some ideas on how to do this here https://github.com/nextauthjs/next-auth/issues/836
